### ISSUE TITLE
Accessibility Optimization 2: Alt tags

### ISSUE THREAD

**Issue by divineaverage** [divineaverage/P4_SiteOptimization#11]
Remove keywords from alt tags and create image descriptions.

**Comment by divineaverage** [comment]
Updated alt text for all images on page, removing SEO keywords and instead describing the image itself for users who cannot see.
![Screen Shot 2022-10-20 at 3 47 27 PM](https://user-images.githubusercontent.com/107645919/197043789-81fcf03a-c9d1-423a-ac54-1cc90bf0360f.png)
